db/db_impl/db_impl.cc

```diff
@@ -3894,19 +3894,21 @@ Status DBImpl::IngestExternalFiles(
       }
     }
     if (status.ok()) {
-      bool should_increment_last_seqno =
-          ingestion_jobs[0].ShouldIncrementLastSequence();
+      int consumed_seqno_count =
+          ingestion_jobs[0].ConsumedSequenceNumbersCount();
 #ifndef NDEBUG
       for (size_t i = 1; i != num_cfs; ++i) {
-        assert(should_increment_last_seqno ==
-               ingestion_jobs[i].ShouldIncrementLastSequence());
+        assert(!!consumed_seqno_count ==
+               !!ingestion_jobs[i].ConsumedSequenceNumbersCount());
+        consumed_seqno_count +=
+            ingestion_jobs[i].ConsumedSequenceNumbersCount();
```

Comment:
This line makes debug build and production build different. Is this a bug?
```
        consumed_seqno_count +=
            ingestion_jobs[i].ConsumedSequenceNumbersCount();
```

Reply:
What is the !! supposed to do?  That is a new operator to me...

Reply:
I think `!!` force converts a integer to a boolean. 0 => 0, others => 1

Reply:
I also posted the question here: https://groups.google.com/g/rocksdb/c/g5LACzCltWI
Thanks in advance!